gamedata.py

```diff
@@ -141,6 +143,22 @@ def floor(self) -> int:
     def floor_time(self) -> int:
         return 0
 
+    @property
+    def cards_obtained(self) -> list[str]:
+        return self.mod_data.get("cardsObtained", [])
```

Comment:
This should check for `has_data` to make sure that `mod_data` is not None (return an empty list if so).